### DISCUSSION TITLE
Is it possible to override the FluentDataGrid methods?

### DISCUSSION THREAD

**Discussion by artur-michalak** [microsoft/fluentui-blazor#2231] [Q&A] [answered]
Is it possible to inherit from FluentDataGrid?
It has a RefreshDataAsync method, which I would like to override by:

```c#
public new async Task RefreshDataAsync()
{
    Loading = true;
    await base.RefreshDataAsync();
}
```

to trigger the loading indicator every time someone has to interact with the grid.

Thank you for your time :)

**Comment by vnbaaij** [comment, marked as answer]
No, the class and/or methods have not been declared as abstract/virtual so can not be overridden.

**Comment by artur-michalak** [comment]
Will there be such an option in the future?

**Reply by vnbaaij** [reply]
Most probably not
